fools/py/main.py

```
@@ -0,0 +1,217 @@
+def is_prime(n: int) -> bool:
+    """Check if a number is prime."""
+    if n < 2:
+        return False
+    for i in range(2, int(math.sqrt(n)) + 1):
+        if n % i == 0:
+            return False
+    return True
+
+
+def generate_random_string(length=8) -> str:
+    return ''.join(random.choice(string.ascii_letters) for _ in range(length))
```

Ruff (0.15.2)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)